buffalo/misc/aux.py

```diff
@@ -108,30 +108,30 @@ def copy_to_temporary_file(source_path, ignore_lines=0, chunk_size=8192, binary=
             return w.name
 
 
-def psort(path, parallel=-1, field_seperator=' ', key=1, tmp_dir='/tmp/', buffer_mb=1024, output=None):
+def psort(path, parallel=-1, field_seperator=" ", key=1, tmp_dir="/tmp/", buffer_mb=1024, output=None):
     # TODO: We need better way for OS/platform compatibility.
     # we need compatibility checking routine for this method.
-    commands = ['sort', '-n', '-s']
+    commands = ["sort", "-n", "-s"]
     if parallel == -1:
         parallel = psutil.cpu_count()
     if parallel > 0:
-        commands.extend(['--parallel', parallel])
+        commands.extend(["--parallel", parallel])
     if not output:
         output = path
-    commands.extend(['-t', '{}'.format(field_seperator)])
-    commands.extend(['-k', key])
-    commands.extend(['-T', tmp_dir])
-    commands.extend(['-S', '%sM' % buffer_mb])
-    commands.extend(['-o', output])
+    commands.extend(["-t", "{}".format(field_seperator)])
+    commands.extend(["-k", key])
+    commands.extend(["-T", tmp_dir])
+    commands.extend(["-S", "%sM" % buffer_mb])
+    commands.extend(["-o", output])
     commands.append(path)
     try:
-        subprocess.check_output(map(str, commands), stderr=subprocess.STDOUT, env={'LC_ALL': 'C'})
+        subprocess.check_output(map(str, commands), stderr=subprocess.STDOUT, env={"LC_ALL": "C"})
     except Exception as e:
-        log.get_logger().error('Unexpected error: %s for %s' % (str(e), ' '.join(list(map(str, commands)))))
+        log.get_logger().error("Unexpected error: %s for %s" % (str(e), " ".join(list(map(str, commands)))))
         raise
 
 
-def get_temporary_file(root='/tmp/', write_mode='w'):
+def get_temporary_file(root="/tmp/", write_mode="w"):
```

Comment:
윈도우는 지원하지 않지만, OS 상관없이 tmp 디렉토리를 의미할수 있게끔 아래처럼 하면 어떨까요?

```suggestion
def get_temporary_file(root=tempfile.gettempdir(), write_mode="w"):
```

Reply:
단테 혹시 이거 별도 PR로 나눠도 괜찮을까요?

Reply:
지금 언급해주신 것들 중요한게 많아 보이긴 하는데, 뭔가 PR 규모가 너무 커지고 있는 것 같습니다